TPGSup/devTPG300.proto

```diff
@@ -48,7 +48,7 @@ getFunctionMacro {
 }
 
 setFunction {
-    out "SP%(\$1)s,%(\$1\$2).1E,%(\$1\$3).1E,%(\$1\$4)d";
+    out "SP%(\$1)s,%(\$1\$2).1E,%(\$1\$3).1E,%(\$1\$4)d%#/E(\+|-)0(\d)/E\1\2/";
```

Comment:
Could we add a comment above this line explaining what is going on? Otherwise this is going to confuse future developers, I think...